[2024-09-21T12:21:52.165+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_saver_v01.fetch_vk_data scheduled__2024-09-21T03:00:00+00:00 [queued]>
[2024-09-21T12:21:55.695+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_saver_v01.fetch_vk_data scheduled__2024-09-21T03:00:00+00:00 [queued]>
[2024-09-21T12:21:55.732+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-09-21T12:21:55.740+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2024-09-21T12:21:55.753+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-09-21T12:22:00.143+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): fetch_vk_data> on 2024-09-21 03:00:00+00:00
[2024-09-21T12:22:00.340+0000] {standard_task_runner.py:55} INFO - Started process 13494 to run task
[2024-09-21T12:22:00.493+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_saver_v01', 'fetch_vk_data', 'scheduled__2024-09-21T03:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/dag_saver.py', '--cfg-path', '/tmp/tmp7khpb9nw']
[2024-09-21T12:22:00.511+0000] {standard_task_runner.py:83} INFO - Job 95: Subtask fetch_vk_data
[2024-09-21T12:22:19.714+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_saver_v01.fetch_vk_data scheduled__2024-09-21T03:00:00+00:00 [running]> on host 16da3808dee2
[2024-09-21T12:23:07.777+0000] {local_task_job.py:159} INFO - Task exited with return code Negsignal.SIGKILL
[2024-09-21T12:23:11.711+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
